src/doc/book/structs.md

```diff
@@ -117,6 +117,28 @@ fn main() {
 }
 ```
 
+We can initializing a data structure (struct, enum, union) with named fields, by writing `fieldname` as a shorthand for `fieldname: fieldname`. This allows a compact syntax for initialization, with less duplication:
```

Comment:
It seems wrong to me.
I would write:
Initialization of a data structure can be simplified if fields of the data structure are initialized with variables which has same names as the fields.